contracts/raffle/Makefile.toml

```diff
@@ -76,6 +76,14 @@ script = '''
 script = '''
   for PLAYER in 1 2 3
   do
+    soroban contract invoke \
+      --id $(cat /tmp/contract_id.txt) \
+      --source player_$PLAYER \
+      --network ${network} \
```

Comment:
I have suspects that maybe if we try this in `testnet` , it could be already solved. Soroban is already available in testnet. I am unsure if both environments are maintained with latest Soroban version. 

Should we try on testnet ?

Reply:
The same problem is on testnet...